src/main/java/frc/robot/RobotContainer.java

```diff
@@ -20,25 +24,37 @@
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 
+import static edu.wpi.first.units.Units.Degrees;
+
 
 public class RobotContainer {
-  
+
   private final DriveSubsystem driveSubsystem = new DriveSubsystem();
 
+  // I think you should use the april tags and get the pose from that. Then you can have a command for every tag
```

Comment:
I like the idea of using a command for each april tag, I was planning on adding that soon. I don't know how useful accessing the exact pose of the tags is though, because we don't want to move on top of them, we want to be some position next to them which could vary from tag to tag. The way I thought was best to do this was to have a position input that we adjust as needed for each tag command. Is there some benefit to using the april tag positions instead? I would think that we would need to still end up putting in arbitrary constants specifying how far away to be